Customer Support Procedures/LKE-v4.1-migration.md

```diff
@@ -0,0 +1,131 @@
+<!-- omit in toc -->
+# Linkurious Enterprise migration to v4.1 guidelines
+
+If you are planning to migrate your Linkurious Enterprise instance from an older version to v4.1,
+this document will provide you extra details of what you need to take care or do for a successful upgrade.
+
+<!-- omit in toc -->
+# Table of contents
+- [General major changes](#general-major-changes)
+  - [Requirements](#requirements)
+  - [Improved support for resources migration](#improved-support-for-resources-migration)
+  - [Resources ownership](#resources-ownership)
+  - [Simplified log configuration](#simplified-log-configuration)
+  - [Security updates](#security-updates)
+- [Dropped Support section](#dropped-support-section)
+
+# Requirements
+
+Updating from Linkurious Enterprise v2.0 or older to v4.1 is not supported.
+If you are updating from Linkurious Enterprise v2.0 or older, please first update to v3.1 or v4.0 before updating to v4.1.
+
+We strongly recommend backing-up your SQL database and your installation folder before updating Linkurious Enterprise.
+
+# General major changes
+
+## Improved support for resources migration
+
+Universally unique identifiers (UUID) are 128-bit labels that look like `c7b30a93-207b-417e-b4f5-4f6e45818595`.
+A UUID uniquely identifies a resource, no matter its kind and the Linkurious Enterprise instance it resides in.
+
+UUIDs have been added to Linkurious enterprise's resources:
+- Custom groups
+- Static queries & query templates
+- Custom actions
+- Node grouping rules
+- Spaces
+- Alerts & alert folders
+
+
+***Impacted clients:***
+
+Everyone using the Configuration Migration Plugin.
+
+***Benefits:***
+
+This solves issues caused by renamed resources, allowing more reliable synchronizations between Linkurious Enterprise instances.
+
+***Impacts:***
+
+Static queries and query templates can be referred to by their identifier in various parts of Linkurious Enterprise:
```

Comment:
let's avoid making the separation between statich and query templates since we also removed this separation from the UI and doc. We can just say "Queries"

Reply:
Sure, it's done :+1: